### ISSUE TITLE
Embedding 3-Valued Parameters from float32 to int8

### ISSUE THREAD

**Issue by yukiharada1228** [yukiharada1228/BitNetb158#2]
I am facing a challenge in embedding 3-valued parameters, stored in float32 format as -1, 0, or 1, into int8. While the data range and resolution remain unchanged, I still need to carefully consider the appropriate conversion method to ensure accuracy is maintained.

**Comment by yukiharada1228** [comment]
If self.beta exists, then it would be advisable to utilize Triton for bit matrix operations, wouldn't it?

**Comment by yukiharada1228** [comment]
I've managed to make it work! Now, we need to be able to build the model solely from quantized data

**Comment by yukiharada1228** [comment]
One approach for saving and loading quantized weights
```
model = bitresnet18b158(num_classes=10)
model.eval()

for module in model.modules():
    if isinstance(module, BitLinearb158):
        module.convert_weights_to_packed()
```
```
import torch

torch.save(model.state_dict(), "packed_weights.pth")

packed_weights = torch.load("packed_weights.pth")

for name, parameter in model.named_parameters():
    if name in packed_weights:
        parameter.data.copy_(packed_weights[name])
```
